aws/README.md

```diff
@@ -0,0 +1,63 @@
+# Deploy to AWS (Beta)
+
+> **Note:** The AWS deployment template is still in **BETA** phase. You may encounter failures during the deployment. In that case, please let us know the issue.
+
+This template will create a fully-working IPSec/L2TP VPN server on AWS (Amazon Web Service). Please make sure to check the [pricing details](https://aws.amazon.com/ec2/pricing/on-demand/) of Virtual Machine on EC2 before starting the launch sequence.
+
+You can also use `t2.micro` instance as your server for your deployment, which is free of charge within the first year since your AWS account is registered. For more information on AWS free usage tier, go to [this page](https://aws.amazon.com/free/).
+
+## Available Customization Parameters:
+
+- AWS EC2 Instance Type
+- OS for your VPN Server (Ubuntu16.04, Ubuntu18.04 or Debian9-stretch)
+- Your VPN username
+- Your VPN password
+- IPSec PSK (pre-shared key)
+
+> When choosing your username and password, do not enter special characters like `" ' \`.
+
+Make sure to do this with an **AWS ROOT ACCOUNT** or an **IAM ACCOUNT** with **ADMINISTRATION PRVILEGE**. AWS Cloudformation requires the template file to be store on AWS S3, while a [copy](./cloudformation-template-ipsec) is available on Github. If you want to make any contributions to this template, make sure to contact me so that I can update the template on S3.
```

Comment:
As an alternative, we can let the user download the template file to their local directory, then visit https://console.aws.amazon.com/cloudformation/home#/stacks/new and upload the template file they downloaded before proceeding. This way, we can keep the template in this repository instead of on Amazon S3. What do you think?

Reply:
That way works. I will create another section telling people how to do that.